src/resources/nfts/index.ts

```diff
@@ -61,25 +40,20 @@ export function useLegacyNFTs({ address }: { address: string }) {
   );
   const isImportedWallet = walletAddresses.includes(address);
 
-  const { data, error, fetchNextPage, hasNextPage, isFetchingNextPage, isFetching } = useInfiniteQuery({
+  const { data, error, isFetching } = useQuery({
     queryKey: nftsQueryKey({ address }),
-    queryFn: async ({ pageParam }) => {
-      const [simplehashResponse, polygonAllowlist] = await Promise.all([fetchSimpleHashNFTs(address, pageParam), fetchPolygonAllowlist()]);
-      const { data, nextCursor } = simplehashResponse;
-      const newNFTs = filterSimpleHashNFTs(data, polygonAllowlist).map(nft => simpleHashNFTToUniqueAsset(nft, address));
-      return {
-        data: newNFTs,
-        nextCursor,
-      };
+    queryFn: async () => {
+      const queryResponse = await arcClient.getNFTs({ walletAddress: address });
```

Comment:
this query uses simplehash's nft model. the nfts are still converted into `UniqueAsset`s client side bc we ultimately want to overhaul `UniqueAsset` so I don't want to contaminate the arc repo with all that messy logic